ansible/roles/real-postfix/tasks/main.yml

```diff
@@ -0,0 +1,89 @@
+---
+- name: Install postfix
+  dnf:
+    name:
+      - postfix
+      - postfix-ldap
+      - postfix-pcre
+      - procmail
+      - spamass-milter-postfix
+      - postfix-perl-scripts
+    state: present
+  notify: reload postfix
+- name: Create /etc/systemd/system/postfix.service.d/
+  file:
+    path: /etc/systemd/system/postfix.service.d/
+    state: directory
+- name: Delay postfix startup until network is online
+  copy:
+    dest: /etc/systemd/system/postfix.service.d/scripts-postfix-network-online.conf
+    content: |
+      [Unit]
+      Wants=network-online.target
+      After=network-online.target
+- name: Install postfix configuration files
+  copy:
+    src: postfix/
+    dest: /etc/postfix/
+  notify: reload postfix
+- name: Install postfix LDAP configuration
+  template:
+    src: "postfix/{{ item }}.j2"
+    dest: /etc/postfix/{{ item }}
+  loop:
+    - authorized-submit-users-ldap.cf
+    - mailbox-command-maps-ldap.cf
+    - virtual-alias-domains-ldap.cf
+    - virtual-alias-maps-ldap-reserved.cf
+    - virtual-alias-maps-ldap.cf
+    - virtual-alias-maps-relay-ldap.cf
+    - virtual-alias-maps-relay-user-ldap.cf
+    - virtual-alias-maps-relay-user-suffix-ldap.cf
+  notify: reload postfix
+- name: Install main.cf
+  template:
+    src: main.cf.j2
```

Comment:
The main reason is just that this was pre-existing and I didn't want to change unrelated bits of code. You're right that this could be moved. Do you want me to do that cleanup as part of this PR?